lib/handler/configurator/proxy.c

```diff
@@ -142,6 +142,52 @@ static int on_config_ssl_cafile(h2o_configurator_command_t *cmd, h2o_configurato
     return ret;
 }
 
+static int on_config_ssl_session_cache(h2o_configurator_command_t *cmd, h2o_configurator_context_t *ctx, yoml_t *node)
+{
+    struct proxy_configurator_t *self = (void *)cmd->configurator;
+    size_t i;
+
+    switch (node->type) {
+    case YOML_TYPE_SCALAR:
+        if (strcasecmp(node->data.scalar, "OFF") == 0) {
+            /* disabled */
+            self->vars->session_cache.capacity = 0;
+        } else if (strcasecmp(node->data.scalar, "ON") == 0) {
+            /* use default values */
```

Comment:
Does this work if for an example the feature is turned off at the global level and then set to `ON` at the host level?